models/turbine_models/tests/sd_test.py

```diff
@@ -230,13 +251,17 @@ def testExportUnetModel(self):
         )
         err = utils.largest_error(torch_output, turbine)
         assert err < 9e-5
+        if UPLOAD_IR:
```

Comment:
like how is this code ever reached?

Reply:
If we are uploading IR, we don't exit() and return the blob name. So, we have to manually raise this exit to make the test happy.